.github/workflows/python-package.yml

```diff
@@ -15,25 +15,25 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [2.7]
+        python-version: [3.6, 3.7, 3.8]
 
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - uses: actions/cache@v2
```

Comment:
Is there a reason to stop caching dependencies?

Reply:
I have switched to the default github action for testing python packages, This is the reason, why this was changed

Reply:
The default Python GitHub Action provides a good starting template for a Python project. In my experience on other projects, that template usually needs to be modified/improved for real-world projects. Even the main GitHub docs provide examples of useful changes to the default template. For example, here's their suggested change to support caching: https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-python#caching-dependencies.

Looking at the current builds in GitHub, they run fast enough. So we don't necessarily _need_ caching of the Python dependencies. But having caching in there doesn't hurt either, so I was surprised to see you take it out.